src/app/chip-zcl-zpro/command-encoder/encoder.cpp

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
git mv is equivalent to mv & git add

git uses a heuristic, if more than a certain % (default 50%) of lines are the same, it's considered a rename. Since this file was substantially rewritten, it's not detected as a rename.